editor/project_manager.cpp

```diff
@@ -2869,8 +2879,8 @@ ProjectManager::ProjectManager() {
 
 	SceneTree::get_singleton()->get_root()->connect("files_dropped", callable_mp(this, &ProjectManager::_files_dropped));
 
-	// Define a minimum window size to prevent UI elements from overlapping or being cut off
-	DisplayServer::get_singleton()->window_set_min_size(Size2(750, 420) * EDSCALE);
+	// Define a minimum window size to prevent UI elements from overlapping or being cut off.
+	DisplayServer::get_singleton()->window_set_min_size(Size2(480, 350) * EDSCALE);
```

Comment:
At this size, things are clipping in the Asset Library Projects tab. If we combine this PR with #61215, then we can get down to 520px wide, so I suggest changing this to 520px.

```suggestion
	DisplayServer::get_singleton()->window_set_min_size(Size2(520, 350) * EDSCALE);
```